docs/en/ingest-management/tab-widgets/add-fleet-server/content.asciidoc

```diff
@@ -52,20 +57,65 @@ For more information about these settings, see <<fleet-settings>>.
 [role="screenshot"]
 image::images/add-fleet-server.png[In-product instructions for adding a {fleet-server}]
 
+//QUESTION: Why does the Fleet Server install command include the --url flag
+//when Production mode is selected)? It's not there in Quick Start mode.
+//I thought --url was only required when you are enrolling subsequent agents in
+// Fleet Server.
+
+//QUESTION: How do cloud users handle the problem with certs if they select
+//Production mode? 
```

Comment:
This is a good question. I think Elastic Cloud uses public CAs so the user does not need to supply certs when enrolling agents. I think the same would be true if the self-managed fleet server is using either public certs, like we offer on Elastic cloud, or private certs if the host has the CA installed at the OS level. I'm less sure if we can handle the case where they are using a cloud-based fleet server and a self-managed fleet server with a private CA. Should we say this use case is not supported at the moment @nimarezainia @blakerouse?

Reply:
@mostlyjason  This is potentially confusing to users on multiple levels. Since users need to select either Quick Start or Production, Cloud users will see the command that shows all the certs flags, even if they aren't required to pass them. 

It feels like these options shouldn't be available when running Kibana on cloud (unless I'm missing something here).

Reply:
That's a good point. I'd like to confirm with engineering if the above is correct. If so, we should probably remove those options on cloud.